devlab/tests/filtering_utils.py

```diff
@@ -69,6 +77,34 @@ def filter_tenants(self, src_data_list):
                 src_data_list.pop(index)
         return [src_data_list, popped_tenant_list]
 
+
+class MigrationUtils(object):
+
+    def __init__(self):
+        self.username_for_ssh = 'cirros'
```

Comment:
It would be better to inherit username/password from config.py file 

ssh_check_user = 'cirros'
ssh_check_password = 'cubswin:)'

( I did the same here: https://github.com/MirantisWorkloadMobility/CloudFerry/pull/497/files )